js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts

```diff
@@ -202,6 +204,12 @@ export const MESSAGE_NAME =
 export const MESSAGE_TOOL_CALLS =
   `${SemanticAttributePrefixes.message}.${MessageAttributePostfixes.tool_calls}` as const;
 
+/**
+ * The id of the tool call
```

Comment:
nit:

```suggestion
 * The id of a tool call on a tool role message
```

or somethign like that to differntiate from the one below